src/server/list_family_test.cc

```diff
@@ -420,6 +420,35 @@ TEST_F(ListFamilyTest, LPop) {
   EXPECT_THAT(resp, ArgType(RespExpr::NIL));
 }
 
+// Add LMPOP
+TEST_F(ListFamilyTest, LMPOP) {
+  // Set up initial conditions for the test
+  Run({"rpush", "mylist", "a", "b", "c", "d"});
+
+  // Test LMPOP for popping two elements from the left
+  auto resp = Run({"LMPOP", "mylist", "2"});
```

Comment:
According to the documentaion this is how the command should look like
LMPOP numkeys key [key ...] <LEFT | RIGHT> [COUNT count]
you need to follow this

Reply:
also I dont see where you added the implementation for the command, parsing the relevant args